### DISCUSSION TITLE
My General Midi Drum Kit

### DISCUSSION THREAD

**Discussion by NoAliby** [hydrogen-music/hydrogen#1510] [General]
My General Midi Drum Kit

This is a simple General Midi Drum Kit 
Primarily objective is for importing GM midi patterns into Hydrogen. 
It is Fully General Midi Compliant with 53 Notes (47x GM1, 6x GM2)
------------------------
All GM1 Samples (Except # 65, 66, 78, 79)  are from:
  FreePats General MIDI sound set 
   - Version 2021-03-29 
   - https://freepats.zenvoid.org/SoundSets/general-midi.html 
   - GNU General Public License version 3 or later, with a special exception 
   - https://freepats.zenvoid.org/licenses.html#GPL_exception 

All GM2 Samples (Plus GM1# 65, 66, 78, 79) are from:
  Free Wave Samples
   - https://freewavesamples.com 
   - All of the samples recorded by Jason Champion, 
   - Distributed as freeware and may be used royalty-free.
   - https://freewavesamples.com/about-us-license 
--------------------------
Let me know what you think

**Comment by NoAliby** [comment]
[My General Midi Kit v1.0.zip](https://github.com/hydrogen-music/hydrogen/files/8159218/My.General.Midi.Kit.v1.0.zip)

V1.0
- First Release

**Comment by theGreatWhiteShark** [comment]
Hey @NoAliby,

Could you apply the changes you did to the BeatBuddy kit to this one as well? Then I'll publish it too.
